xml/ha_cluster_lvm.xml

```diff
@@ -573,6 +571,20 @@ Login to [iface: default, target: iqn.2010-03.de.&wsI;:san1, portal: &wwwip;,326
       Free  PE / Size       254 / 1016,00 MB
       VG UUID               UCyWw8-2jqV-enuT-KH4d-NXQI-JhH3-J24anD</screen>
     </step>
+    <step>
+     <para>
+      Check the shared state of the volume group with the command <command>vgs</command>:
+     </para>
+     <screen>&prompt.root;<command>vgs</command>
+  VG       #PV #LV #SN Attr   VSize     VFree  
+  vgshared   1   1   0 wz--ns 1016.00m  1016.00m</screen>
+     <para>
+      The <literal>Attr</literal> column shows the volume attributes. In this example,
+      the volume group is writable (<literal>w</literal>),
+      resizeable (<literal>z</literal>), the allocation policy is normal (<literal>n</literal>),
+      and it is a shared resource (<literal>s</literal>).
+      See the <command>man</command> page with the command <command>man vgs</command> for details.</para>
```

Comment:
```suggestion
      See the <command>man</command> page of <command>vgs</command> for details.</para>
```

Reply:
According to https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-other-external
;)